process/smartContract/process.go

```diff
@@ -79,33 +81,34 @@ type scProcessor struct {
-	PubkeyConv                          core.PubkeyConverter
-	ShardCoordinator                    sharding.Coordinator
-	ScrForwarder                        process.IntermediateTransactionHandler
-	TxFeeHandler                        process.TransactionFeeHandler
-	EconomicsFee                        process.FeeHandler
-	TxTypeHandler                       process.TxTypeHandler
-	GasHandler                          process.GasHandler
-	GasSchedule                         core.GasScheduleNotifier
-	BuiltInFunctions                    process.BuiltInFunctionContainer
-	TxLogsProcessor                     process.TransactionLogProcessor
-	BadTxForwarder                      process.IntermediateTransactionHandler
-	DeployEnableEpoch                   uint32
-	BuiltinEnableEpoch                  uint32
-	PenalizedTooMuchGasEnableEpoch      uint32
-	RepairCallbackEnableEpoch           uint32
-	StakingV2EnableEpoch                uint32
-	ReturnDataToLastTransferEnableEpoch uint32
-	SenderInOutTransferEnableEpoch      uint32
-	EpochNotifier                       process.EpochNotifier
-	IsGenesisProcessing                 bool
-	ArwenChangeLocker                   process.Locker
+	VmContainer                                 process.VirtualMachinesContainer
+	ArgsParser                                  process.ArgumentsParser
+	Hasher                                      hashing.Hasher
+	Marshalizer                                 marshal.Marshalizer
+	AccountsDB                                  state.AccountsAdapter
+	BlockChainHook                              process.BlockChainHookHandler
+	PubkeyConv                                  core.PubkeyConverter
+	ShardCoordinator                            sharding.Coordinator
+	ScrForwarder                                process.IntermediateTransactionHandler
+	TxFeeHandler                                process.TransactionFeeHandler
+	EconomicsFee                                process.FeeHandler
+	TxTypeHandler                               process.TxTypeHandler
+	GasHandler                                  process.GasHandler
+	GasSchedule                                 core.GasScheduleNotifier
+	BuiltInFunctions                            process.BuiltInFunctionContainer
+	TxLogsProcessor                             process.TransactionLogProcessor
+	BadTxForwarder                              process.IntermediateTransactionHandler
+	DeployEnableEpoch                           uint32
+	BuiltinEnableEpoch                          uint32
+	PenalizedTooMuchGasEnableEpoch              uint32
+	RepairCallbackEnableEpoch                   uint32
+	StakingV2EnableEpoch                        uint32
+	ReturnDataToLastTransferEnableEpoch         uint32
+	SenderInOutTransferEnableEpoch              uint32
+	IncrementSCRNonceInMultiTransferEnableEpoch uint32
+	EpochNotifier                               process.EpochNotifier
+	IsGenesisProcessing                         bool
```

Comment:
can you move the bool arg in the last position of the struct? for memory alignment reasons.

Reply:
done